frontend/src/components/HacknightsParticipants.vue

```diff
@@ -65,6 +65,9 @@
               <v-icon>mdi-account-outline</v-icon>
             </v-list-item-avatar>
             <v-list-item-title v-text="item.github"></v-list-item-title>
+            <v-btn v-on:click="onDeleteParticipant(item)">
```

Comment:
How would you like this?
```suggestion
            <v-btn icon v-on:click="onDeleteParticipant(item)">
```